app/javascript/packages/webauthn/enroll-webauthn-device.ts

```diff
@@ -48,36 +75,7 @@ async function enrollWebauthnDevice({
       challenge,
       rp: { name: window.location.hostname },
       user,
-      pubKeyCredParams: [
-        {
-          type: 'public-key',
-          alg: -7, // ECDSA w/ SHA-256
-        },
-        {
-          type: 'public-key',
-          alg: -35, // ECDSA w/ SHA-384
-        },
-        {
-          type: 'public-key',
-          alg: -36, // ECDSA w/ SHA-512
-        },
-        {
-          type: 'public-key',
-          alg: -37, // RSASSA-PSS w/ SHA-256
-        },
-        {
-          type: 'public-key',
-          alg: -38, // RSASSA-PSS w/ SHA-384
-        },
-        {
-          type: 'public-key',
-          alg: -39, // RSASSA-PSS w/ SHA-512
-        },
-        {
-          type: 'public-key',
-          alg: -257, // RSASSA-PKCS1-v1_5 w/ SHA-256
-        },
-      ],
+      pubKeyCredParams: SUPPORTED_ALGORITHMS.map((alg) => ({ alg, type: 'public-key' })),
```

Comment:
Maybe we don't even need the `SUPPORTED_ALGORITHMS` constant at all if we do:
```suggestion
      pubKeyCredParams: Object.values(COSEAlgorithm).map((alg) => ({ alg, type: 'public-key' })),
```

Reply:
ok and it turns out `Object.values()` lost the typing so this doesn't work either

Reply:
```suggestion
      pubKeyCredParams: (Object.values(COSEAlgorithm) as COSEAlgorithm[]).map((alg) => ({ alg, type: 'public-key' })),
```